### PR TITLE
Update about-snowflake-native-app.md

### PR DESCRIPTION
**Pull request by amychen1776**
## What are you changing in this pull request and why?
I'm not quite sure if this goes on the about the native app page or on the setup page and I'm very open to discussion here!

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud-integrations/about-snowflake-native-app.md [hunk 1]
```diff
@@ -45,3 +45,6 @@ If you're interested, please [contact us](matilto:sales_snowflake_marketplace@db
 
 ## Support
 If you have any questions about the dbt Snowflake Native App, you may [contact our Support team](mailto:dbt-snowflake-marketplace@dbtlabs.com) for help. Please provide information about your installation of the Native App, including your dbt Cloud account ID and Snowflake account identifier. 
+
+## Limitations
+- The Native app does not support dbt Cloud accounts that have [IP Restrictions](/docs/cloud/secure/ip-restrictions} enabled.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/about-snowflake-native-app.md]:
```suggestion
- The Native app does not support dbt Cloud accounts with [IP Restrictions](/docs/cloud/secure/ip-restrictions] enabled. 
```

**Comment by mirnawong1** [comment]
hey @amychen1776 ! i think this could certainly live in both areas and happy to create a snippet for you here. however, wanted to double check whether we should provide a 'why' or whether this will be supported in the future?

**Comment by amychen1776** [comment]
@mirnawong1 Perhaps this fits in the FAQs section on the setup page as a snippet? Would you be able to help me on that?
I could also use your help with the wording. The reason why it's not supported is Snowflake does not have static IPs for SPCS so it's difficult for us to know what to allowlist in dbt Cloud for access. We can look at alternative ways to do this but it'll take some work to scope.

**Comment by mirnawong1** [comment]
happy to @amychen1776 ! will sort it now for you

**Comment by mirnawong1** [comment]
hey @amychen1776 , @matthewshaver mentioned that he's been chatting with you on this so passing the baton to him to help out here! thank you and sorry for the confusion folks! 🙏

**Comment by amychen1776** [comment]
No worries! Thank you so much for your help 🙏

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**